File: apidemo/sensordemo/environmentalsensordemo/src/main/java/com/gdkdemo/sensor/environmental/service/SensorValueQueryUtil.java
package com.gdkdemo.sensor.environmental.service;

import com.gdkdemo.sensor.environmental.cp.data.sensorvalue.SensorValueData.SensorValues;


public final class SensorValueQueryUtil
{

    public static final String[] PROJECTION = new String[] {
            SensorValues._ID,             // 0
            SensorValues.GUID,       // 1
            SensorValues.TYPE,       // 2
            SensorValues.ACCURACY,       // 3
            SensorValues.VAL0,       // 4
            SensorValues.VAL1,       // 5
            SensorValues.VAL2,       // 6
            SensorValues.VAL3,       // 7
            SensorValues.TIMESTAMP,       // 8
            SensorValues.CREATEDTIME,       // 9
            SensorValues.MODIFIEDTIME,       // 10
    };

    public static final int COLUMN_INDEX_GUID = 1;
    public static final int COLUMN_INDEX_TYPE = 2;
    public static final int COLUMN_INDEX_ACCURACY = 3;
    public static final int COLUMN_INDEX_VAL0 = 4;
    public static final int COLUMN_INDEX_VAL1 = 5;
    public static final int COLUMN_INDEX_VAL2 = 6;
    public static final int COLUMN_INDEX_VAL3 = 7;
    public static final int COLUMN_INDEX_TIMESTAMP = 8;
    public static final int COLUMN_INDEX_CREATEDTIME = 9;
    public static final int COLUMN_INDEX_MODIFIEDTIME = 10;




}

File: xamarindemo/sensordemo/EnvironmentalSensorDemo/Service/SensorValueQueryUtil.cs
using System;

namespace EnvironmentalSensorDemo
{
	public class SensorValueQueryUtil
	{
		public static string[] PROJECTION = new string[] {
			Android.Provider.BaseColumns.Id,             // 0
			SensorValueData.SensorValues.GUID,       // 1
			SensorValueData.SensorValues.TYPE,       // 2
			SensorValueData.SensorValues.ACCURACY,       // 3
			SensorValueData.SensorValues.VAL0,       // 4
			SensorValueData.SensorValues.VAL1,       // 5
			SensorValueData.SensorValues.VAL2,       // 6
			SensorValueData.SensorValues.VAL3,       // 7
			SensorValueData.SensorValues.TIMESTAMP,       // 8
			SensorValueData.SensorValues.CREATEDTIME,       // 9
			SensorValueData.SensorValues.MODIFIEDTIME,       // 10
		};

		public static int COLUMN_INDEX_GUID = 1;
		public static int COLUMN_INDEX_TYPE = 2;
		public static int COLUMN_INDEX_ACCURACY = 3;
		public static int COLUMN_INDEX_VAL0 = 4;
		public static int COLUMN_INDEX_VAL1 = 5;
		public static int COLUMN_INDEX_VAL2 = 6;
		public static int COLUMN_INDEX_VAL3 = 7;
		public static int COLUMN_INDEX_TIMESTAMP = 8;
		public static int COLUMN_INDEX_CREATEDTIME = 9;
		public static int COLUMN_INDEX_MODIFIEDTIME = 10;


	}
}

